### DISCUSSION TITLE
linguist stopped ignoring vendored paths

### DISCUSSION THREAD

**Discussion by GaidaiIgor** [github-linguist/linguist#5372] [Classification] [answered]
Hello,
In my [repo](https://github.com/IgorGayday/SpectrumSDT) I have `.gitattributes` file with the following content:
> PES_examples/* linguist-vendored
> libs/* linguist-vendored

which is supposed to tell linguist to exclude the content of the `PES_examples` and `libs` folders from statistics.

This was working as expected until it just stopped working at some point for no apparent reason. Now if you look at language statistics in my repo it is mostly influenced by language of libraries. Is there something wrong on my side or is this a bug?

**Comment by GaidaiIgor** [comment, marked as answer]
Alright, I found out that I need double asterisk to make it recursive now.

**Reply by unknwon** [reply]
Ended up in here too... 😞 thank you to saving my day!

**Reply by lildude** [reply]
https://github.com/github/linguist/discussions/5329#discussioncomment-623406 has more info.
